src/tsolvers/egraph/Enode.h

```diff
@@ -266,13 +207,7 @@ class EnodeAllocator : public RegionAllocator<uint32_t>
     void free(ERef eid)
```

Comment:
Above this, is the `const Enode & dbg` used and if it's not should we remove it?

Reply:
It does not seem to be. I am removing it.

Reply:
Removed.